AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/MAX/Native Ads/ALMAXManualNativeAdViewController.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  ALMAXManualNativeAdViewController.swift
+//  AppLovin MAX Demo App - Swift
+//
+//  Created by Billy Hu on 1/20/22.
+//  Copyright © 2022 AppLovin. All rights reserved.
+//
+
+import UIKit
+import Adjust
+import AppLovinSDK
+
+class ALMAXManualNativeAdViewController: ALBaseAdViewController, MAAdRevenueDelegate
+{
+    @IBOutlet weak var nativeAdContainerView: UIView!
+    
+    private var nativeAdLoader: MANativeAdLoader!
+    private var nativeAdView: MANativeAdView!
+    private var nativeAd: MAAd?
+    
+    // MARK: View Lifecycle
+    
+    override func viewDidLoad()
+    {
+        super.viewDidLoad()
+        
+        let nativeAdViewNib = UINib(nibName: "NativeCustomAdView", bundle: Bundle.main)
+        
+        nativeAdView = nativeAdViewNib.instantiate(withOwner: nil, options: nil).first! as! MANativeAdView?
+        
+        let adViewBinder = MANativeAdViewBinder(builderBlock: { (builder) in
+            builder.titleLabelTag = 1001
+            builder.advertiserLabelTag = 1002
+            builder.bodyLabelTag = 1003
+            builder.iconImageViewTag = 1004
+            builder.optionsContentViewTag = 1005
+            builder.mediaContentViewTag = 1006
+            builder.callToActionButtonTag = 1007
+        })
+        nativeAdView.bindViews(with: adViewBinder)
+        
+        nativeAdLoader = MANativeAdLoader(adUnitIdentifier: "YOUR_AD_UNIT")
+        nativeAdLoader.nativeAdDelegate = self
+        nativeAdLoader.revenueDelegate = self
+    }
+    
+    deinit
+    {
+        cleanUpAdIfNeeded()
+        
+        nativeAdLoader.nativeAdDelegate = nil
+        nativeAdLoader.revenueDelegate = nil
+    }
+    
+    // MARK: IB Actions
+    
+    @IBAction func showAd()
+    {
+        cleanUpAdIfNeeded()
+
+        nativeAdLoader.loadAd(into: nativeAdView)
+    }
+    
+    func cleanUpAdIfNeeded()
```

Comment:
Same comment as well, move this above the `// MARK`. Same comment for the template counterparts.